test/rebalancer/receiving_bucket.test.lua

```diff
@@ -43,6 +43,7 @@ box.space.test4:count()
 box.space.test5:count()
 
 vshard.storage.bucket_send(1, util.replicasets[2], {timeout = 10})
+wait_bucket_is_collected(1)
```

Comment:
This line is not related to the patch, it's just a possible flakiness (and it's reproduced). The bucket can be already garbage collected, let's fix it alongside.